app/views/setup/_cluster_size_panel.html.slim

```diff
@@ -0,0 +1,19 @@
+.panel.panel-default
+  .panel-heading
+    h3.panel-title Cluster size
+  .panel-body.cluster-size-selector
+    .col-md-8
+      .form-group
+        label for="cloud_cluster_instance_count" Number of instances
+        = form.text_field :instance_count, data: {slider_min: 3, slider_value: 5, slider_max: 50, slider_scale: "logarithmic", slider_tooltip: "always", slider_tooltip_position: "bottom", slider_ticks: [3, 5, 10, 20, 50], slider_ticks_snap_bounds: 0.5}
+    .col-md-2.text-right
+      .form-group.cluster-cpu-count
```

Comment:
Is `form-group` really necessary?

Reply:
It makes the bootstrap structure simpler, and avoids duplicating the related CSS.

Reply:
But using this class here only adds an unnecessary margin-bottom that makes the layout a bit strange because there's more space in the bottom than in the top. You can see the difference  when you choose "Other type", this element gets hidden and the spacing gets the same as the top.

Reply:
Fixed in d15b5f90.

Reply:
👍